src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
Let's separate the issues here:

1. Changing the [deprecated] string to [Obsolete].
2. Showing attribute information in Quick Info tooltips.

For 1. I agree with @DustinCampbell that this change is gratuitous, and perhaps even misleading.  There is a variety of other information we show in Quick Info (like exception text, xml doc summaries, async usages, etc) that *does not* come from attributes.  The fact that a symbol is deprecated is a similar piece of information.  Changing it to look more like the attribute may make people think that we *do* show all attributes in Quick Info, so it's actually a good thing that it doesn't look like the attribute.

For 2, that is already tracked by #711, but that PR doesn't address that feature request.  I now think that we should probably close this PR and #780, and think about #711 differently.

Sorry for the confusion on this one.